src/js/controller/tracks-loader.ts

```diff
@@ -4,17 +4,22 @@ import { ajax } from 'utils/ajax';
 import { localName } from 'parsers/parsers';
 import srt from 'parsers/captions/srt';
 import dfxp from 'parsers/captions/dfxp';
-import { composePlayerError, convertToPlayerError, ERROR_LOADING_CAPTIONS } from 'api/errors';
+import { composePlayerError, convertToPlayerError, ERROR_LOADING_CAPTIONS, PlayerError } from 'api/errors';
```

Comment:
`PlayerError` should be imported as a type on a new line:
`import type { PlayerError } from 'api/errors';`